sdk/servicebus/service-bus/test/streamingReceiverSessions.spec.ts

```diff
@@ -343,7 +343,7 @@ describe("Streaming with sessions", () => {
           async processMessage(msg: ReceivedMessageWithLock) {
             return msg.abandon().then(() => {
               abandonFlag = 1;
-              if (receiver.isReceivingMessages()) {
+              if ((receiver as any)._isReceivingMessages()) {
```

Comment:
Can we not do `receiver as SessionReceiverImpl` here?

Reply:
Can do.

Reply:
Done